addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2630,6 +2630,15 @@ def test_out_invoice_switch_out_refund_2(self):
             'amount_untaxed' : self.move_vals['amount_untaxed'],
         })
 
+    def test_out_invoice_switch_out_refund_3(self):
+        # Test to check that when switching from out_invoice to out_refund the fiscal position is kept
+        move = self.init_invoice('out_invoice')
+        move.fiscal_position_id = self.fiscal_pos_a.id
+
+        move.action_switch_move_type()
+
+        self.assertRecordValues(move, [{'fiscal_position_id': self.fiscal_pos_a.id}])
```

Comment:
Sorry, missed it before, as we check only one field, I think `assertEqual` is enough

```suggestion
        self.assertEqual(move.fiscal_position_id, self.fiscal_pos_a)
```